File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { RedTideInformationSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await RedTideInformationSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("red-tide-information_sdk")

describe("RedTideInformationSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
